docs/trackmate_ui.rst

```diff
@@ -0,0 +1,103 @@
+Analyze OMERO timelapse images using the TrackMate User Interface
+=================================================================
+
+In this example we open in Fiji an image stored in an OMERO server and use `TrackMate <https://imagej.net/TrackMate>`_ to analyze it.
+
+Description
+-----------
+
+In this section, we show how to use TrackMate via its User Interface.
+The manual steps are essential to determine the suitable parameters to analyze the images. Note that when using the TrackMate User Interface, the generated tracks **cannot** be saved as ROIs into OMERO.server.
+
+Setup
+-----
+
+- Install Fiji on the local machine with the OMERO.insight-ij plugin. The installation instructions can be found in the `Fiji guide <https://omero-guides.readthedocs.io/en/latest/fiji/docs/installation.html>`_.
+
+Resources
+---------
+
+-  We use an artificial track image \ https://samples.fiji.sc/FakeTracks.tif.
+
+
+Step-by-Step
+------------
+
+We will use the manual workflow to explain the steps of the script below.
```

Comment:
There is no script below here. Maybe this sentence is stale and should be removed ?